Problem-1.py

```diff
@@ -0,0 +1,61 @@
+"""
+## Problem1 (https://leetcode.com/problems/subarray-sum-equals-k/)
+Given an array of integers and an integer k, you need to find the total number of continuous subarrays whose sum equals to k.
+
+Example 1:
+
+Input:nums = [1,1,1], k = 2
+
+Output: 2
+
+Note:
+
+The length of the array is in range [1, 20,000].
+The range of numbers in the array is [-1000, 1000] and the range of the integer k is [-1e7, 1e7].
+
+// Time Complexity : O(n)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No, Shazmaan explained really well
+
+"""
+# brute force
+def subarray_sum(nums,k):
```

Comment:
What is the time complexity for this solution?

Reply:
O(n^3) or O(n^2) I am a little confused.